### ISSUE TITLE
Error formatting file when no bugs

### ISSUE THREAD

**Issue by mark-lazarides** [Barteling/STweep#69]
**Describe the bug**
When formatting some DUTs in a library project, STWeep (4.04) is throwing errorr that: Error formatting file, code contains parsing errors and cannot be parsed.

**Expected behavior**
The same code currently compiles in an active PLC project - I assume STWeep should not report this error and format the file

**STweep version info**
"STweep for TwinCAT 4.01"

**IDE version info**
"Microsoft Visual Studio Community 2019 Version 16.11.16"

**Additional context**
```PASCAL
{attribute 'pack_mode' := '1'}
TYPE stKL306x_ChannelStatus:
STRUCT
	Underrange 	: BIT;
	Overrange 	: BIT;
	Limit1Bit_0 : BIT;
	Limit1Bit_1 : BIT;
	Limit2Bit_0 : BIT;
	Limit2Bit_1 : BIT;
	Error		: BIT;
	{attribute 'hide'}	
	pack_0_7	: BIT;
END_STRUCT
END_TYPE;
`

**Comment by Barteling** [comment]
Got it, please remove the semicolon after 'END_TYPE' for STweep to work.

I'll update my parser to possibly expect a semicolon after END_TYPE

**Comment by mark-lazarides** [comment]
That fixed that one! Another has popped up (sorry, I'm updating an old libary). Exactly the same error, on this:
```PASCAL
PROPERTY FINAL InstanceName : T_MaxString; // returns the fully scoped variable name
```

**Comment by Barteling** [comment]
Problem seems to be with the FINAL operator, didnt expect that on a property..
Will be fixed

**Comment by Barteling** [comment]
Fixed in 4.1.0-Beta, could you let me know this worked?
https://www.stweep.com/beta-releases/
